pkg/serverstate/serverstate.go

```diff
@@ -177,7 +177,8 @@ type Interface interface {
 	TaskPut(*pb.Task) error
 	TaskGet(*pb.Ref_Task) (*pb.Task, error)
 	TaskDelete(*pb.Ref_Task) error
-	TaskList() ([]*pb.Task, error)
+	TaskList(*pb.ListTaskRequest) ([]*pb.Task, error)
+	GetJobsByTaskRef(*pb.Task) (*pb.Job, *pb.Job, *pb.Job, error)
```

Comment:
Do we normally have a `Get` prefix? I don't recall. I thought idiomatic Go was `JobsByTaskRef` instead.

Reply:
Addressed in 3270d63103b7197eef42d6d88928cd04bd927d75